apps/hashdotdev/src/pages/roadmap/technology-tree.tsx

```diff
@@ -205,7 +205,13 @@ export const TechnologyTree: FunctionComponent = () => {
           completion status.
         </Typography>
       </Container>
-      <Container>
+      <Container
+        sx={{
+          maxWidth: {
+            lg: 1500,
```

Comment:
@nonparibus as you suggested in Figma I've made the technology tree wider than the paragraph elements (up to `1500px`). Let me know if you'd like me to adjust it to something else.